src/hidimstat/conditional_feature_importance.py

```diff
@@ -188,3 +193,71 @@ def _permutation(self, X, group_id):
         return self._list_imputation_models[group_id].sample(
             X_minus_j, X_j, n_samples=self.n_permutations
         )
+
+
+def cfi(
+    estimator,
+    X,
+    y,
+    cv=KFold(n_splits=5, shuffle=True, random_state=0),
+    groups: dict = None,
+    var_type: str = "auto",
+    method: str = "predict",
+    loss: callable = root_mean_squared_error,
+    n_permutations: int = 50,
+    imputation_model_continuous=None,
+    imputation_model_categorical=None,
+    categorical_max_cardinality: int = 10,
+    k_best=None,
+    percentile=None,
+    threshold=None,
+    threshold_pvalue=None,
+    random_state: int = None,
+    n_jobs: int = 1,
+):
+    methods = CFI(
+        estimator=estimator,
+        method=method,
+        loss=loss,
+        n_permutations=n_permutations,
+        imputation_model_continuous=imputation_model_continuous,
+        imputation_model_categorical=imputation_model_categorical,
+        categorical_max_cardinality=categorical_max_cardinality,
+        random_state=random_state,
+        n_jobs=n_jobs,
```

Comment:
Same comment as above, I would suggest parallelizing over the CV, not the features.